src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/ActiveEditorContextTracker.cs

```diff
@@ -56,57 +52,42 @@ public int GetProjectName(uint itemid, out string pbstrProjectName)
             return HResult.OK;
         }
 
-        public bool IsActiveEditorContext(IWorkspaceProjectContext context)
+        public bool IsActiveEditorContext(string contextId)
         {
-            Requires.NotNull(context, nameof(context));
+            Requires.NotNullOrEmpty(contextId, nameof(contextId));
 
-            if (_contexts.TryGetValue(context, out string projectContextId))
-            {
-                return StringComparers.WorkspaceProjectContextIds.Equals(ActiveIntellisenseProjectContext, projectContextId);
-            }
+            if (!_contexts.Contains(contextId))
+                throw new InvalidOperationException($"'nameof{contextId}' has not been registered or has already been unregistered");
```

Comment:
Nit: This looks odd... is nameof not supposed to be there, supposed be inside the braces, or both?